pkgs/by-name/mu/museum/package.nix

```diff
@@ -7,14 +7,14 @@
 
 buildGoModule rec {
 
-  version = "photos-v0.9.5";
+  version = "v0.9.5";
   pname = "museum";
 
   src = fetchFromGitHub {
     owner = "ente-io";
     repo = "ente";
     sparseCheckout = [ "server" ];
-    rev = version;
+    rev = "photos-v${version}";
```

Comment:
I think you should either remove the `v` prefix from here, or from the `version` attribute.

Reply:
fixed.